### PR TITLE
SASS-1508 - Professional fees and subscriptions radio button page

### PR DESCRIPTION
**Pull request by ina-tu**
### Description
added radio button page for professional fees and subscriptions benefits

Add a link to the relevant story in Jira
[SASS-1449](https://jira.tools.tax.service.gov.uk/browse/SASS-1449)

##### Before Reviewing
- [x]  Have you pulled the branch down?
- [x]  Have you assigned yourself to the PR?
- [x]  Have you moved the task to “in review” on JIRA?
- [x]  Have you checked to ensure all dependencies are up to date?
- [x]  Have you checked to ensure its been rebased against the current version of master?

##### Whilst Reviewing
- [x]  Have you run the tests?
- [x]  Have you run the journey tests?
- [x]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?

##### After Reviewing
- [x]  Have you checked for merge conflicts or any changes in the current master that may affect the current pull request? i.e. does it need another rebase?
- [ ]  Have you checked to make sure there are no builds in the pipeline before you merge?
- [ ]  Have you moved the task to “in pipeline” on Jira?

##### Before creating PR
- [x]  Have you run the tests?
- [x]  Have you run the journey tests? (where applicable)
- [x]  Have you addressed warnings where appropriate?
- [x]  Have you rebased against the current version of master?
- [x]  Have you checked code coverage isn’t lower than previously?

##### After PRs been raised
- [x]  Have you checked the PR Builder passes?

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -772,10 +772,17 @@ benefits.entertainmentBenefitAmount.error.overMaximum.agent = Your client’s en
 benefits.entertainmentBenefitAmount.error.invalidFormat.individual = Enter the amount you got for entertainment in the correct format
 benefits.entertainmentBenefitAmount.error.invalidFormat.agent = Enter the amount your client got for entertainment in the correct format
 
+benefits.professionalSubscriptions.title.individual = Did your employer cover costs for any professional fees or subscriptions?
+benefits.professionalSubscriptions.title.agent = Did your client’s employer cover costs for any professional fees or subscriptions?
+benefits.professionalSubscriptions.paragraphText.individual = Your employer may have covered fees you must pay to be able to do your job. This includes annual subscriptions to approved professional bodies that are relevant to your work.
```
[the remark this answers is not in the document]
Comment (ina-tu) [reply conf/messages:777]: there are a few render the page tests in the ispec that check the content (title check, text on page, etc.)

File: conf/messages.cy [hunk 2]
```diff
@@ -772,11 +772,17 @@ benefits.entertainmentBenefitAmount.error.overMaximum.agent = Your client’s en
 benefits.entertainmentBenefitAmount.error.invalidFormat.individual = Enter the amount you got for entertainment in the correct format
 benefits.entertainmentBenefitAmount.error.invalidFormat.agent = Enter the amount your client got for entertainment in the correct format
 
+benefits.professionalSubscriptions.title.individual = Did your employer cover costs for any professional fees or subscriptions?
+benefits.professionalSubscriptions.title.agent = Did your client’s employer cover costs for any professional fees or subscriptions?
+benefits.professionalSubscriptions.paragraphText.individual = Your employer may have covered fees you must pay to be able to do your job. This includes annual subscriptions to approved professional bodies that are relevant to your work.
+benefits.professionalSubscriptions.paragraphText.agent = Your client’s employer may have covered fees they must pay to be able to do their job. This includes annual subscriptions to approved professional bodies that are relevant to their work.
+benefits.professionalSubscriptions.error.individual = Select yes if your employer covered costs for any  professional fees or subscriptions
```
[the remark this answers is not in the document]
Comment (ina-tu) [reply conf/messages.cy:779]: same as above

[the remark this answers is not in the document]
Comment (georgievh) [reply conf/messages.cy:779]: There is extra space here: ' any  professional' if we wanted to remove it.

[the remark this answers is not in the document]
Comment (ina-tu) [reply conf/messages.cy]: done

**Pull request merged**